### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by IjayAbby**
## Project requirements.
- Build a todo list using Javascript that deletes, add, update and checks a todo. 
- Use Webpack to build the project.
- Host the project on GitHub Pages.
- Use ESlint and Stylelint  linters for code quality.

### CODE REVIEW

File: src/modules/dragDrop.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+export const dragStart = (e) => {
+  e.currentTarget.style.opacity = '0.5';
+
+  dragElement = e.currentTarget;
+
+  e.dataTransfer.effectAllowed = 'move';
+  e.dataTransfer.setData('text/html', e.currentTarget.innerHTML);
+};
+export const dragEnd = (e) => {
```
Comment (LOctavio) [inline src/modules/dragDrop.js:11]: https://github.com/IjayAbby/To-do-list-code-review/blob/05e05054e976faf5488b97a1b01aa174674d859c/src/modules/dragDrop.js#L11-L17

If your arrow functions have just one line inside is better to put everything in one line.

File: src/modules/dragDrop.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  e.currentTarget.style.opacity = '1';
+};
+
+export const dragOver = (e) => {
+  e.preventDefault();
```
Comment (LOctavio) [inline src/modules/dragDrop.js:16]: Finaly, don't forget to add the linters

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+import {
+  dragStart, dragEnd, dragOver, dragDrop,
+} from './modules/dragDrop';
+import { checkBoxStatus, changeStyleTask } from './modules/checkStatus';
+import { Task, inputTask, addTask } from './modules/addtask';
+import {
+  clearCompletedTasks, removeAllIcon, removeAllItems, removeCompletedItem, removeSelectedItem,
+} from './modules/remove';
+
+let toDoTasks = [
```
Comment (LOctavio) [inline src/index.js:13]: https://github.com/IjayAbby/To-do-list-code-review/blob/05e05054e976faf5488b97a1b01aa174674d859c/src/index.js#L13

Use const instad let when creating variables

Comment (LOctavio) [inline src/index.js:13]: would be better you use const instead of let when you create an array

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+const getTaskData = () => {
+  if (localStorage.getItem('TaskData') != null) {
+    toDoTasks = JSON.parse(localStorage.getItem('TaskData'));
+  }
+};
+
+const setData = () => {
+  localStorage.setItem('TaskData', JSON.stringify(toDoTasks));
+};
```
Comment (LOctavio) [inline src/index.js]: you can write everything in one line if your block has just one line of code inside

File: src/modules/renderTasks.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    );
+  } else {
+    listContainer.insertAdjacentHTML(
+      'beforeend',
+      `<li data-id="${task.index}" draggable="true" class="item">
+      <input class="checkbox" type="checkbox" data-id="${task.index}">
+      <label class="text-task" data-id="${task.index}" contenteditable=true>${task.description}</label><br>
+      <i class="fas fa-trash-alt remove" data-id="${task.index}"></i>
+      <i class="fas fa-ellipsis-v icon-item"></i>
+      </li>
```
Comment (LOctavio) [inline src/modules/renderTasks.js:23]: You are repeating your code

**Comment by LOctavio** [comment]
Finally, don't forget to add linters to your project.

File: src/modules/renderTasks.js (hunk 5 above)
Comment (IjayAbby) [reply src/modules/renderTasks.js:23]: Thanks, how would i best refactor?

File: src/index.js (hunk 3 above)
Comment (IjayAbby) [reply src/index.js:13]: For const I cannot reassign it

**Review by LOctavio — APPROVED** [review]
Good job. 

status: approved.

**Pull request merged**